apps/desktop/src/features/workspace/firstRunRoles.ts

```
@@ -0,0 +1,47 @@
+/** Closed first-run role choices that map onto admitted analysis role IDs. */
+
+export const FIRST_RUN_WHOLE_BAND_ROLE_FOCUS = [
+  "bass-guitar",
```

ESLint

[error] Missing JSDoc comment.

(jsdoc/require-jsdoc)